src/SFA.DAS.AssessorService.Api.Types/Models/AO/OrganisationData.cs

```diff
@@ -11,5 +13,7 @@ public class OrganisationData
         public string Postcode { get; set; }
         public string CompanyNumber { get; set; }
         public string CharityNumber { get; set; }
+        public DateTime? FinancialDueDate { get; set; }
+        public bool? FinancialExempt { get; set; }
```

Comment:
nullable bool is a bit smelly for what is effectively a yes/no answer.

Reply:
it's a yes/no/don't know answer, so valid here